build/custom-ci-Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM openshift/origin-release:golang-1.14
+RUN yum -y install epel-release --disablerepo=epel && yum clean all
+RUN yum -y install make
+RUN go get -u github.com/onsi/ginkgo/ginkgo
```

Comment:
According to https://onsi.github.io/ginkgo/ we may also need:
```
go get github.com/onsi/gomega/...
```

I would add it to be safe